deepchem/models/torch_models/layers.py

```diff
@@ -3743,3 +3743,71 @@ def forward(
             P = pair_features
 
         return [A, P]
+
+class Highway(nn.Module):
+    """
+        Highway layer used in TextCNN model.
```

Comment:
Formatting here is off. Please make sure to check alignment

Reply:
I'll fix the spacing issues, I want to ask for rest formating is yapf and mypy enough or should I do mannual check ?

Reply:
Consider testing: yapf, flake8, doctest, pytest, mypy for each .py file you modify